renovate.json

```
@@ -0,0 +1,26 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "rebaseWhen": "behind-base-branch",
+    "dependencyDashboard": true,
```

Remove unless we find them really helpful?
```suggestion
    "dependencyDashboard": false,
```